stories/tableOfContents/tableOfContents.handlebars

```diff
@@ -1,10 +1,10 @@
 <nav class="toc {{class}}" role="navigation" aria-label="{{ariaLabel}}">
-  {{#if title}}
-  <a class="toc__title" href="/digital-preservation-policy/">{{title}}</a>
-  {{/if}}
   <div class="toc__list">
+    {{#if title}}
```

Comment:
What's the logic in dropping this inside the `toc__list` class, and is there any point in retaining that wrapping div?

Reply:
I dropped it inside to more easily adjust styles for a group of a title and associated list items together. But I've dropped that div and the `toc__list` entirely now to keep it simple. If we need to add divs later for layout, we can do that when we implement.